pylint/checkers/utils.py

```diff
@@ -852,3 +852,20 @@ def is_registered_in_singledispatch_function(node):
             return decorated_with(func_def, singledispatch_qnames)
 
     return False
+
+
+def infer_sequence_with_starred_expressions(seq_node):
```

Comment:
I would rather see this inference support brought in astroid. What do you think?